src/Sulu/Bundle/SecurityBundle/Resources/translations/admin.en.json

```diff
@@ -2,6 +2,8 @@
     "sulu_security.roles": "User roles",
     "sulu_security.user_name": "Username",
     "sulu_security.enable_user": "Enable User",
+    "sulu_security.lock_user": "Lock User",
+    "sulu_security.user_locked": "User locked",
```

Comment:
How about just using this translation? I think it is always confusing, if a toggler changes the text, because it is not clear if it is the current state, or the new one when you click on it. If we just use "User locked" both times it is clearer to me that the user is no locked if the toggler is deactivated.